docs/docs/technology/mantis/tools.md

```diff
@@ -0,0 +1,55 @@
+# Tools
+## Development Progress
+As of November 15, 2023 MANTIS is in mainnet testing. Details of this testing period are as follows:
+
+Composable is running MANTIS and the Composable Virtual Machine (CVM) on mainnet. Osmosis, Neutron, and the Composable Cosmos chain will be accessible for cross-chain operations on the mainnet testing. The flow and components of the testing are as follows:
+- We will create faucets for people to use
+- There is a place where solvers can connect to the mainnet testing over an RPC
+- There is a place where problems can be submitted by users over an RPC
+
+Additional public details of this testing can be found on [this notion page](https://www.notion.so/composablefoundation/Banksy-Testnet-4-72b7f8317bcf406e9059ae258733eb31), which includes specifications and setup/joining instructions. The code repo is available on [this GitHub page](https://github.com/notional-labs/composable-networks/tree/main/banksy-testnet-4).
```

Comment:
i do not develop on testnet and cannot, until  somebody maintains IBC connection alive, they dead all the time.

I do mainnet and devnet.

Reply:
If there are not operating bridges there is no MANTIS. Bridges first, MANTIS second.

People struggle mainnet bridge, so I have doubts to run testnets operational is easy.

Reply:
which ibc connection on testnet do you need, we can help get it set up

Reply:
removed this content for now

Reply:
for any networks which are subject for mantis. also not only get, but maintain.  and for each restnet indexer i use for pools and gas prices.